### ISSUE TITLE
Question / Comment:Issue with LIGATURES

### ISSUE THREAD

**Issue by moka1309** [pymupdf/PyMuPDF#497]
Hi again,
I am facing another issue, while extracting text, words contains chars ff, fi, etc,. are having space and if there are both ff's then one f is missing.
Examples:

- "Pfizer" turns into ---> "Pﬁ zer"
- "affect" turns as ----> "af ect"
- "effective" turns ----> "ef ective"
- "different" ----> "dif erent".

I read about prevent ligatures, and written code like below
`blocks = page.getText("blocks", flags=1+2+8)`
But not successful.

**Comment by moka1309** [comment]
From this [file](https://github.com/pymupdf/PyMuPDF/files/4557385/DAURISMO.Patient.Brochure_Post-MLR.copy.pdf) I am extracting blocks. But when ever I ecounterd words like 
```
"Pfizer"
"affect"
"effective"
"different"
```
I am getting them in output as 

```
"Pﬁ zer"
"af ect"
"ef ective"
"dif erent"
```

**Comment by JorjMcKie** [comment]
Confirmed - reproduced.
I (PyMuPDF) can nothing do about it. It happens inside MuPDF. Obviously some fonts or how a PDF is technically created causes this type of problem. I know that they have a logic to recognize if text appears twice with a minimal difference of the respective bboxes. In that case it is assumed to be a case of "faked bold" and the text is extracted only once.

**Comment by moka1309** [comment]
Thanks for the info. I am closing this thread.
